AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -274,6 +274,14 @@ + (void)setDelegate:(_Nullable id<MSCrashesDelegate>)delegate {
 
 #pragma mark - Service initialization
 
++ (void)load {
+  [MSAppCenterUserDefaults addKeysToMigrate:@{
+    @"kMSCrashesIsEnabledKey" : @"MSAppCenterCrashesIsEnabledKey",              // MSCrashes
+    @"MSAppDidReceiveMemoryWarning" : @"MSAppCenterAppDidReceiveMemoryWarning", // MSCrashes
+    @"MSUserConfirmation" : @"MSAppCenterUserConfirmation"                      // MSCrashes
```

Comment:
`MSAppCenterUserConfirmation` could be less generic, for instance: `MSAppCenterCrashesUserConfirmation`